common/reviews/api/rush-lib.api.md

```diff
@@ -312,10 +312,10 @@ export type ResolutionStrategy = 'fewer-dependencies' | 'fast';
 
 // @public
 export class Rush {
-    static launch(launcherVersion: string, arg: ILaunchOptions): void;
+    static launch(launcherVersion: string, arg: ILaunchOptions, cb?: () => void): void;
```

Comment:
This is a public API.  If you want to add `cb`, it should be:
- spelled out as a proper English word
- documented 
- moved onto `ILaunchOptions`

It seems like an internal detail, though -- it's doubtful that it really needs to be part of the  public contract for  `launch()`

Reply:
I will remove this.